### ISSUE TITLE
Annoying bug on iOS

### ISSUE THREAD

**Issue by GorgonUK** [vcmi/vcmi#4138]
**Describe the bug**
Whatever you do in the game, the white bar at the bottom keeps appearing and the disappearing. Is there a fix for it? When the white bar is visible, you can't see some of your resources on iOS.

**The Screenshots**
![image](https://github.com/vcmi/vcmi/assets/59618079/f2f90f25-38e1-4c00-ab16-8b28cce555fe)

![image](https://github.com/vcmi/vcmi/assets/59618079/4349809b-c428-4901-b92a-3c9b10601dc6)

**Version**
 - OS: iOS 17
 - Version: 1.5.2

**Comment by kambala-decapitator** [comment]
this white bar is a feature of iOS and can't be hidden completely.

probably the best we can do is "cut" game screen in the bottom that it doesn't overlap with the bar.

**Comment by GorgonUK** [comment]
> this white bar is a feature of iOS and can't be hidden completely.
> 
> probably the best we can do is "cut" game screen in the bottom that it doesn't overlap with the bar.

What is causing that to happen and what is the most sensible fix for it?

**Comment by kambala-decapitator** [comment]
that bar is the replacement for the Home button on modern iOS devices. In most apps it's always displayed on screen, but an app can choose to hide it when you don't interact with the screen (that's what you're seeing). App developers don't have any other way to affect the bar.

> probably the best we can do is "cut" game screen in the bottom that it doesn't overlap with the bar.

this would be the fix, I don't see any other way

**Comment by GorgonUK** [comment]
Okay, understood. I don't think people want to lose screen real estate, especially those who play it on iPhones.

Thanks for clarifying the above.

**Comment by kambala-decapitator** [comment]
Will check how it looks if applied to iPad. For iPhone it's definitely a bad idea.

**Comment by peterpaulis** [comment]
i guess this issue can be closed, since this doesn't happen on iPad18 and iPadOS26, on the 1.6

**Comment by kambala-decapitator** [comment]
yes, there was an adjustment to move game screen a bit, the offset is configurable in the launcher

**Comment by GorgonUK** [comment]
Yes, the white bar is now tucked in on iOS 26 and it is no longer an issue.
